spec/unit/type/k5login_spec.rb

```diff
@@ -69,6 +77,19 @@ def resource(attrs = {})
           end
         end
 
+        context "with selinux" do
+          subject { resource(:content => "daniel@EXAMPLE.COM\n",).retrieve }
+          it "should return correct values based on SELinux state" do
+            Puppet::Type::K5login::ProviderK5login.any_instance.stubs(:selinux_support?).returns true
++            Puppet::Type::K5login::ProviderK5login.any_instance.stubs(:get_selinux_current_context).returns "user_u:role_r:type_t:s0"
```

Comment:
Looks like a stray `+` from the diff made it in here at the beginning of the line.